src/email/logs.html

```diff
@@ -27,33 +27,33 @@
             <legend>{{ 'email.logging.search' | translate }}</legend>
             <div class="form-group">
                 <label for="logsSubject" class="placeholder">{{'email.logging.search' | translate }}</label>
-                <input id="logsSubject" type="text" ng-model="search.subject" />
+                <input id="logsSubject" type="text" ng-model="search.subject" ng-disabled="isLoading" />
```

Comment:
Let's not make these changes one-by-one in the markup — we will want this functionality on every page, so lets write an input directive in common, and add an event watcher that disables inputs on 'loading' or 'loading.start'